src/main.py

```diff
@@ -100,6 +100,56 @@ async def get_handler(req):
     return response.json(json_response['results']['bindings'])
 
 
+@app.route('/list', methods=['GET'])
+async def get_handler(req):
+    sparql_url = environ.get('SPARQL_URL')
+    print(check_query_args_list(req.query_args))
+    if not check_query_args_list(req.query_args):
+        return response.json({'message': 'Wrong query parameters!'}, status=400)
+
+    print("continuing")
+    name = req.query_args[0][1]
+    lastname = req.query_args[1][1]
+
+    query = """
+        PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+        PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+        PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+        PREFIX persoon: <http://data.vlaanderen.be/ns/persoon#>
+        PREFIX mandaat:<http://data.vlaanderen.be/ns/mandaat#>
+            SELECT DISTINCT ?listName ?listNumber
+            WHERE {
+                ?person persoon:gebruikteVoornaam "%s";
+                foaf:familyName "%s".
+                ?list rdf:type mandaat:Kandidatenlijst;
+                mandaat:heeftKandidaat ?person;
+                skos:prefLabel ?listName;
+                mandaat:lijstnummer ?listNumber.
+        }""" % (name, lastname)
+    query_response = requests.get(
+        sparql_url,
+        params={
+            "default-graph-uri": "http://api.sep.osoc.be/mandatendatabank",
+            "format": "json",
+            "query": query
+        }
+    )
+    json_response = json.loads(
+        query_response.content.decode('utf-8')
+    )
+    return response.json(json_response['results']['bindings'])
+
+
+def check_query_args_list(query_args):
```

Comment:
I think this will bug out if you pass arguments in the wrong other (`?lastname=a&name=b`). You might want to you `res.args` instead, which returns a dictionary (so you can do `res.args['name']`)